### PR TITLE
DS-722 | @ericbakenhus | Add title highlighting

### PR DESCRIPTION
**Pull request by ericbakenhus**
# READY FOR REVIEW

# Summary
- Adds query highlighting for event titles

# Review By (Date)
- Retro

# Criticality
- Low

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to /events/discovery
3. Search for something
4. Verify any matches in titles are highlighted

# Associated Issues and/or People
- DS-722

### CODE REVIEW

File: src/components/events-discovery/components/Event/Event.jsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ import { EventContent } from './EventContent';
  * @property {boolean} [isMinimal]
  * @property {boolean} [isBigHeadline]
  * @property {number} [headingLevel]
+ * @property {EventHit} [hit]
```
Comment (sherakama) [inline src/components/events-discovery/components/Event/Event.jsx:30]: @ericbakenhus are you manually adding these or is something in your IDE helping you?

Comment (ericbakenhus) [reply src/components/events-discovery/components/Event/Event.jsx:30]: [VS Code supports JSDoc](https://code.visualstudio.com/docs/languages/javascript#_jsdoc-support), so I do have some template and autocomplete support there.

Comment (sherakama) [reply src/components/events-discovery/components/Event/Event.jsx:30]: Ty

**Review by sherakama — COMMENTED** [review]
Gtg

**Pull request merged**